### PR TITLE
Manpage updates V2

### PR DESCRIPTION
**Pull request by poettering**
Incorporating all suggested changes from the #348 review (modulo the "the" suggestions).

I figure we should ask a native speaker about the "the" removals, and then fix up all man pages if necessary and not just the new ones added.

### CODE REVIEW

**Comment by poettering** [comment]
@keszybz hmm, the semaphore build fails due to some issue with the python manpage code, any idea what might be going on?

There's probably something wrong with "make update-man-list" too. The file names passed to it are generated from MANPAGES, But MANPAGES is actually what we read from the generate Makefile-man.am file, hence we feedback what we generated before. THat's certainly not right? any idea?

**Comment by poettering** [comment]
BTW, asked native english speaker @blixtra about the "the" issue, and he said both with and without is fine, though he'd probably put a "the" in the sentences...

**Comment by keszybz** [comment]
> the semaphore build fails due to some issue with the python manpage code, any idea what might be going on?

Any chance we could build this with python3? It would report the nested exception. If not, the script could be patched to provide that on it's own, but I would prefer not to clutter the code with workarounds.

> There's probably something wrong with "make update-man-list" too. The file names passed to it are generated from MANPAGES, But MANPAGES is actually what we read from the generate Makefile-man.am file, hence we feedback what we generated before. THat's certainly not right? any idea?

It got broken in https://github.com/systemd/systemd/issues/215.

**Comment by keszybz** [comment]
Wrt. to the "the": OK, in this case "the functions described below" is fairly specific, so I guess it is indeed better to include the definite article.

File: man/sd-bus-errors.xml [hunk 1]
```diff
@@ -0,0 +1,309 @@
...
+        authorization, when interactive authorization was not enabled
+        with the
+        <citerefentry><refentrytitle>sd_bus_message_set_allow_interactive_authorization</refentrytitle><manvolnum>3</manvolnum></citerefentry>
+        for the method call message.</para></listitem>
+      </varlistentry>
+    </variablelist>
+  </refsect1>
+
+  <refsect1>
+    <title>Notes</title>
```
Comment (daurnimator) [inline man/sd-bus-errors.xml:287]: This notes section doesn't make sense when in a man page about header defines.

**Comment by zonque** [comment]
The python error is due to an unmatched `variablelist`of class `sd-bus-errors`. The python magic tries to insert these things into an auto-generated man page but fails. Dropping the class argument fixes the issue.

**Comment by zonque** [comment]
I fixed all minor nits of this thing in #520. Please review. Closing this one in favor.

**Pull request closed without merging**